NEPS/Hacks/Misc.cpp

```diff
@@ -38,6 +38,17 @@
 
 #include <numeric>
 
+struct custom_cmd
+{
+	float			viewangles;
+	float			forwardmove;
+	float			sidemove;
+	float			upmove;
+};
+bool hasShot;
+int qp_count;
+std::vector<custom_cmd>usercmds_for_quickpeek;

```

Comment:
There is no practical reason to do so but just to keep the overall coding style, you should make these vars static